mne/io/eeglab/eeglab.py

```diff
@@ -285,7 +311,7 @@ def __init__(self, input_fname, montage, preload=False, eog=(),
     def _read_segment_file(self, data, idx, fi, start, stop, cals, mult):
         """Read a chunk of raw data"""
         _read_segments_file(self, data, idx, fi, start, stop, cals, mult,
-                            dtype=np.float32)
+                            dtype=np.float32, eeglab=True)
```

Comment:
I'd call it `stim_channel` or something of that sort. So that it's generic for different readers which may want to use this.